src/graphics/gfx_draw_player.cpp

```diff
@@ -18,12 +18,121 @@
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 
+#include <array>
+
 #include "../globals.h"
 #include "../graphics.h"
 #include "../collision.h"
 #include "../core/render.h"
 #include "../gfx.h"
 
+//! Get left pixel at the player sprite
+int pfrX(const StdPicture& tx, const Player_t& p)
+{
+    // will use internal flags (tx.flags & PLAYER_MODERN and tx.flags & PLAYER_CUSTOM) in future
+    // if tx.flags & PLAYER_CUSTOM, then will use heap-allocated polymorphic CustomData_t* tx.custom_data,
+    // which will include all frame bounding boxes and offsets
+    if(tx.w == 1000)
+        return ((p.Frame * p.Direction + 49) / 10) * 100;
+    else
+    {
```

Comment:
Really, better to add these flags now, even if they will not be set yet. The w==1000 is a freaking heuristic...

Reply:
Okay, for such a block I made the "enable experimental features" zone to don't make this work out of our local debugging builds.